src/components/CountrySelector.tsx

```diff
@@ -1,39 +1,33 @@
-import PropTypes from 'prop-types';
-import React, {useEffect} from 'react';
+import React, {type ForwardedRef, useEffect} from 'react';
 import {View} from 'react-native';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import Navigation from '@libs/Navigation/Navigation';
+import type CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
 import FormHelpMessage from './FormHelpMessage';
 import MenuItemWithTopDescription from './MenuItemWithTopDescription';
-import refPropTypes from './refPropTypes';
 
-const propTypes = {
+type CountrySelectorProps = {
     /** Form error text. e.g when no country is selected */
-    errorText: PropTypes.string,
+    errorText?: string;
 
     /** Callback called when the country changes. */
-    onInputChange: PropTypes.func.isRequired,
+    onInputChange: (value?: string) => void;
 
     /** Current selected country  */
-    value: PropTypes.string,
+    value?: keyof typeof CONST.ALL_COUNTRIES;
 
     /** inputID used by the Form component */
     // eslint-disable-next-line react/no-unused-prop-types
-    inputID: PropTypes.string.isRequired,
-
-    /** React ref being forwarded to the MenuItemWithTopDescription */
-    forwardedRef: refPropTypes,
+    inputID: string;
 };
 
-const defaultProps = {
-    errorText: '',
-    value: undefined,
-    forwardedRef: () => {},
+type CountrySelectorPropsWithForwardedRef = CountrySelectorProps & {
+    forwardedRef: ForwardedRef<View>;
```

Comment:
we should change the logic here , take a look at different [component](https://github.com/Expensify/App/blob/main/src/components/BaseMiniContextMenuItem.tsx) where we are passing refs , so remove `forwardedRef` prop and use `forwardRef` from React